cpu/fe310/Kconfig

```
@@ -0,0 +1,57 @@
+# Copyright (c) 2020 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config CPU_ARCH_RISCV
+    bool
+    select HAS_ARCH_RISCV
+
+config CPU_CORE_RV32M
+    bool
+    select CPU_ARCH_RISCV
+    select HAS_ARCH_32BIT
+
+config CPU_FAM_FE310
+    bool
+    select CPU_CORE_RV32M
+    select HAS_CPU_FE310
+    select HAS_PERIPH_CPUID
+    select HAS_PERIPH_GPIO
+    select HAS_PERIPH_GPIO_IRQ
+    select HAS_PERIPH_PM
+    select HAS_PERIPH_WDT
+    select HAS_CPP
+    select HAS_SSP
+
+config CPU_MODEL_FE310_G000
+    bool
+    select CPU_FAM_FE310
+
+config CPU_MODEL_FE310_G002
+    bool
+    select CPU_FAM_FE310
+
+## Definition of specific features
+config HAS_ARCH_RISCV
+    bool
+    help
+        Indicates that the current CPU has a RISC-V.
+
+## Definition of specific features
+config HAS_CPU_FE310
+    bool
+    help
+        Indicates that a 'fe310' cpu is being used.
+
+config CPU_FAM
+    default "fe310" if CPU_FAM_FE310
+
+config CPU_MODEL
+    default "fe310_g000"  if CPU_MODEL_FE310_G000
+    default "fe310_g002"  if CPU_MODEL_FE310_G002
+
+config CPU
+    default "fe310"
```

Although under normal circumstances this file should only be included when this CPU is used, this may be useful for scripts that go over all the Kconfig files:
```suggestion
    default "fe310" if CPU_FAM_FE310
```